pkg/v1/tkg/clusterclient/clusterclient.go

```diff
@@ -320,6 +320,8 @@ type Client interface {
 	ListCLIPluginResources() ([]cliv1alpha1.CLIPlugin, error)
 	// VerifyCLIPluginCRD returns true if CRD exists else return false
 	VerifyCLIPluginCRD() (bool, error)
+	// IsClassyCluster check whether cluster is ClusterClass based or not
+	IsClassyCluster(clusterName, namespace string) (bool, error)
```

Comment:
nit: IsClassyCluster -> IsClusterClassBased to be consistent with with other parts of the code.